stdlib/public/core/ErrorType.swift

```diff
@@ -155,6 +155,13 @@ public func _stdlib_getErrorEmbeddedNSError<T : Error>(_ x: UnsafePointer<T>)
   return x.pointee._getEmbeddedNSError()
 }
 
+/// FIXME: Quite unfortunate to have both of these.
+@_silgen_name("swift_stdlib_getErrorEmbeddedNSErrorValue")
+public func _stdlib_getErrorEmbeddedNSErrorValue<T : Error>(_ x: T)
```

Comment:
For consistency with other things, could you call the UnsafePointer-taking one `_stdlib_getErrorEmbeddedNSErrorIndirect`, and drop "value" from this one?